### ISSUE TITLE
Błędy Estetyczne (Zbiorczy)

### ISSUE THREAD

**Issue by myeshko1** [foundryvttpl/wfrp4e-core-pl#20]
# Podręcznik Główny (Core)
### Tłumaczenie w skrypcie
- [ ] Brak tłumaczenia przy opcji "Rest & Recover"
<img width="383" height="587" alt="Image" src="https://github.com/user-attachments/assets/47fe9c51-4a8a-414d-8dc5-0918cb4b8ab6" />

### Braki

- [ ] #27
![image](https://github.com/user-attachments/assets/8e8e45b8-440c-411c-9b0e-5088d189e4db)

- [ ] Brak tłumaczenia niektórych poziomów trudności testu:
![image](https://github.com/user-attachments/assets/607190f4-9040-4a8c-9c89-f30eac0db27c)

### Błędy estetyczne
Brak obecnie.

**Comment by silentmark** [comment]
dzięki, ogarnie się w tym tygodniu.

**Comment by GregorHak** [comment]
Postacie z Ekranu Mistrza Gry (czyli Dieter Käsegeier, Kastor Erichson, Stellemar) 
https://github.com/silentmark/wfrp4e-core-pl/pull/19/commits/9013f620a39c9c0a075695de2b078c4229db6eaa

**Comment by GregorHak** [comment]
![obraz](https://github.com/user-attachments/assets/7aec63c6-4f4f-473c-a1b3-39b13babe93a)

 
Te poziomy trudności to są z dodatku WFRP 4ed - Wewnętrzny Wróg - Wróg w Cieniach, jak ktoś nie ma wykupionego modułu Enemy in Shadows to nie ma opcji ich zobaczyć nawet. One są częścią tego dodatkowego modułu.

**Comment by GregorHak** [comment]
**Postacie z Ekranu Mistrza Gry (czyli Dieter Käsegeier, Kastor Erichson, Stellemar) nie mają przetłumaczonych opisów.**
aktualizacja nie naprawiła problemu:
https://github.com/silentmark/wfrp4e-core-pl/issues/29

**Comment by mczuchnowski** [comment]
> powinno być Język (Bretoński)

Nie, nazwy języków zapisuje się po polsku małą literą. Jeśli koniecznie chcemy wielką, to `Język (Bretonia)`.

**Comment by Livithor** [comment]
> > powinno być Język (Bretoński)
> 
> Nie, nazwy języków zapisuje się po polsku małą literą. Jeśli koniecznie chcemy wielką, to `Język (Bretonia)`.

Sprawdź jak jest w oficjalnym tłumaczeniu, bo my tylko kopiujemy. Poza tym, ze względów stylistycznych, często stosuje się wielkie litery niezależnie od zasad pisowni, a wszystkie umiejki i rzeczy mechaniczne w podręczniku są pisane wielką literą.

**Comment by TomasWarchol** [comment]
zamykam, nieaktualne od v14
